src/components/Table/TableHeaderColumn.jsx

```diff
@@ -7,12 +7,18 @@ import styles from './Table.css';
 
 export default ({ column, sortClick, sortField, sortState }) => {
   const tooltipId = uuid.v4();
+  const style = {};
+
+  if (column.center) {
+    style.justifyContent = 'center';
+  }
 
   return (
     <MaterialTableHeaderColumn>
       <div
         className={column.sortFn ? styles.headerCell : styles.headerCellNoSort}
         onClick={() => column.sortFn && sortClick(column.field, sortState, column.sortFn)}
+        style={style}
```

Comment:
I don't understand this comment. The CSS you wrote is already in the css file. What exactly are you proposing?